specs/0119-transactional-emails.md

```diff
@@ -0,0 +1,428 @@
+* Tools such [Pontoon](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/index.html) and [Fluent](https://mozilla-l10n.github.io/localizer-documentation/tools/fluent/index.html) (the advanced translation format used by Mozilla)
+
+**Pontoon Add-on:**   
+You can use the Pontoon Add-on for [Firefox](https://mozilla-l10n.github.io/localizer-documentation/index.html) or [Chrome](https://chromewebstore.google.com/detail/pontoon-add-on/gnbfbnpjncpghhjmmhklfhcglbopagbb) to be instantly notified of any Pontoon notifications right in your browser, as well as some helpful tools to help with proofreading, bug reporting, and fixing. 
+
+**Mozilla community participation guidelines:**  
+As an active member of our community, we’d like you to be aware of the [Mozilla community participation guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) to ensure the wellbeing of all members of the community.
+
+**Email settings:**  
+Finally, if you’re interested in hearing about upcoming events organized for our localization community, be sure to enable email communications in your [settings](https://pontoon.mozilla.org/settings/) so you can be the first to get notified.  
+You can also receive Pontoon notifications in a daily or weekly digest — check out the Notification subscriptions section of your settings.
+
+Thanks,  
+Pontoon Team
+
+### Inactive account: Contributor
+
+#### Trigger
+
+All of the following conditions met:
+
+1. Contributor role   
+2. At least 1 non-ML translation submitted   
+3. No login for 6 months  
+4. Has not received inactive account reminder in the past
+
+#### Template
+
+Subject: Mozilla Pontoon account status  
+Body:  
+Hello! 
+
+It's been a while since we’ve last seen you, but we wanted to thank you for contributing to Mozilla’s community-driven localization efforts at [https://pontoon.mozilla.org/](https://pontoon.mozilla.org/). 
+
+Your contributions have been important in furthering Mozilla’s mission and fostering an open and accessible internet for all readers in your language. We understand that life can get busy, and finding time to contribute to projects isn't always easy. We get it. 
+
+As part of our efforts to keep our localization community active and engaged, we periodically review account activity. We’ve noticed you haven’t signed in to your Pontoon account in the past 6 months. While we completely understand that circumstances change, we wanted to inform you that we may deactivate dormant accounts in the future to ensure the security and efficiency of our platform.
+
+If you’d like to keep your Pontoon account active (we hope you will!) please sign in to Pontoon within the next 30 days by visiting [https://pontoon.mozilla.org/](https://pontoon.mozilla.org/) and click the “Sign in” button in the top right corner.
```

Comment:
I kind of missed this (@Delphine raised it in a separate conversation). Since we're not going to disable accounts, and the sentence before is "we may deactivate", should we remove the `within the next 30 days`?

Reply:
Good point, we can always update this if we implement deactivation.